app/waf/src/controllers/adc.controller.ts

```diff
@@ -28,24 +28,17 @@ export class AdcController {
     public adcRepository: AdcRepository,
   ) {}
 
+  readonly createDesc = 'ADC resource that need to be created';
   @post(prefix + '/adcs', {
     responses: {
-      '200': {
-        description: 'Successfully create ADC resource',
-        content: {'application/json': AdcSchema.adcResponse},
-      },
-      '400': {
-        description: 'Invalid ADC resource',
-        content: {'application/json': AdcSchema.BadRequest},
-      },
-      '422': {
-        description: 'Unprocessable ADC resource',
-        content: {'application/json': AdcSchema.UnprocessableEntity},
-      },
+      '200': Schema.response(Adc, 'adc', 'Successfully create ADC resource'),
```

Comment:
There's no cases till now the key name in response is NOT `entity.modelName`.  case insensitive 

Could you please remove the parameter, and get it within `response` function?

Reply:
Done

Reply:
Done